src/Approvals/SFA.DAS.Approvals/InnerApi/Requests/BulkUploadUpdateLogRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using SFA.DAS.Approvals.Application.Shared.Enums;
+
+namespace SFA.DAS.Approvals.InnerApi.Requests
+{
+    public class BulkUploadUpdateLogRequest
+    {
+        public long? ProviderId { get; set; }
+        public long? LogId { get; set; }
+    }
+}
```

Comment:
sorry, see previous comments about unused code

Reply:
hi paul, removed this as part of removing all Update endpoint